.cookiecutter.json

```diff
@@ -2,9 +2,9 @@
   "_template": "gh:cjolowicz/cookiecutter-hypermodern-python",
   "author": "stilllman",
   "email": "luc_touraille@yahoo.fr",
```

Comment:
Should we change this to HACF ?

Reply:
HACF added

Reply:
Added to `pyproject.toml` authors but not here, it takes the first of the list I think.

But maybe this can stay like this.